features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -33,21 +37,27 @@ import { ExtensionTemplateListInterface } from "@wso2is/admin.template-core.v1/m
 import { isFeatureEnabled } from "@wso2is/core/helpers";
 import { AlertLevels, IdentifiableComponentInterface } from "@wso2is/core/models";
 import { addAlert } from "@wso2is/core/store";
+import { Field, Forms } from "@wso2is/forms";
```

Comment:
do not use `Forms` use `FinalForm` instead.
view this PR for more details, "https://github.com/wso2/identity-apps/pull/6722/files#diff-d94522cb3bcc2277d7af7c0f1c38e4e8d1d7659edb195199b995d045be70b235"